Data/models/Bookmark.swift

```diff
@@ -314,7 +314,8 @@ extension Bookmark {
             }
             
             if let ct = customTitle, !ct.isEmpty {
-                bookmarkToUpdate.customTitle = customTitle
+                bookmarkToUpdate.customTitle = ct
+                bookmarkToUpdate.title = ct
```

Comment:
An updated bookmark from desktop comes with `customTitle` and `title` having the same value. We have to match the behavior